src/puppet.ts

```diff
@@ -289,21 +289,21 @@ export abstract class Puppet extends EventEmitter {
-  public on (event: 'friendship',   listener: (friendshipId: string) => void)                                                           : this
-  public on (event: 'login',        listener: (contactId: string) => void)                                                              : this
-  public on (event: 'logout',       listener: (contactId: string) => void)                                                              : this
-  public on (event: 'message',      listener: (messageId: string) => void)                                                              : this
-  public on (event: 'reset',        listener: (reason: string) => void)                                                                 : this
-  public on (event: 'room-join',    listener: (roomId: string, inviteeIdList: string[], inviterId:  string) => void)                    : this
-  public on (event: 'room-leave',   listener: (roomId: string, leaverIdList:  string[], removerId?: string) => void)                    : this
-  public on (event: 'room-topic',   listener: (roomId: string, newTopic:      string,   oldTopic:   string, changerId: string) => void) : this
-  public on (event: 'room-invite',  listener: (roomInvitationId: string) => void)                                                       : this
-  public on (event: 'scan',         listener: (qrcode: string, status: ScanStatus, data?: string) => void)                              : this
-  public on (event: 'ready',        listener: () => void)                                                                               : this
+  public on (event: 'dong',         listener: (data?: string) => void)                                                                                     : this
+  public on (event: 'error',        listener: (error: string) => void)                                                                                     : this
+  public on (event: 'friendship',   listener: (friendshipId: string) => void)                                                                              : this
+  public on (event: 'login',        listener: (contactId: string) => void)                                                                                 : this
+  public on (event: 'logout',       listener: (contactId: string) => void)                                                                                 : this
+  public on (event: 'message',      listener: (messageId: string) => void)                                                                                 : this
+  public on (event: 'reset',        listener: (reason: string) => void)                                                                                    : this
+  public on (event: 'room-join',    listener: (roomId: string, timestamp: number, inviteeIdList: string[], inviterId:  string) => void)                    : this
```

Comment:
The same:

1. move `timestamp` to the end of the argument list
1. remove unrelated changes. (align)